CRM/Contribute/BAO/Contribution.php

```diff
@@ -1101,13 +1101,14 @@ protected static function cancel($processContributionObject, $memberships, $cont
    *
    * @param array $params
    * @param string $context
-   * @param array $previousContributionStatus
-   * @param string $currentContributionStatus
    *
    * @return bool[]
    *   Return indicates whether the updateFinancialAccounts function should continue & whether this is a refund.
    */
-  private static function updateFinancialAccountsOnContributionStatusChange(&$params, $context, $previousContributionStatus, $currentContributionStatus) {
+  private static function updateFinancialAccountsOnContributionStatusChange(&$params, $context) {
+    $previousContributionStatus = CRM_Contribute_PseudoConstant::contributionStatus($params['prevContribution']->contribution_status_id, 'name');
+    $currentContributionStatus = CRM_Core_PseudoConstant::getName('CRM_Contribute_BAO_Contribution', 'contribution_status_id', $params['contribution']->contribution_status_id);
```

Comment:
@eileenmcnaughton This is wierd - we are using two different pseudoconstant functions to get the statuses - can we switch the other one to getName() as part of this PR?

Reply:
@mattwire can we not scope creep this one - it's moving existing lines

Reply:
@mattwire per comment - I'd rather just move & not change within this Pr